src/main/java/core/basesyntax/service/impl/PurchaseHandlerImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitsStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationHandler;
+
+public class PurchaseHandlerImpl implements OperationHandler {
+    @Override
+    public boolean applyOperation(FruitTransaction fruitTransaction) {
+        boolean isSubstracted = false;
```

Comment:
Why do you need to return boolean value if operation applied? The only way it can be not applied is when some exception appear, but in this case you would not reach return statement anyway.